classes/Link.php

```diff
@@ -1027,22 +1028,23 @@ public function getImageLink($name, $idImage, $type = null, string $extension =
      * Returns a link to a supplier image for display.
      *
      * @param int $idSupplier
-     * @param string|null $type image type (small_default, medium_default, large_default, etc.)
+     * @param string|null $type Image thumbnail name (small_default, medium_default, large_default, etc.)
```

Comment:
@Hlavtox as I had conflicts here, I want to have your review to be sure I didn't make mistake.

Reply:
@nicosomb It doesn't matter really, but go with the new one. :-)